lib/src/data_objects/fetch_qualified_segments_response.dart

```diff
@@ -0,0 +1,32 @@
+/// **************************************************************************
+/// Copyright 2023, Optimizely, Inc. and contributors                        *
+///                                                                          *
+/// Licensed under the Apache License, Version 2.0 (the "License");          *
+/// you may not use this file except in compliance with the License.         *
+/// You may obtain a copy of the License at                                  *
+///                                                                          *
+///    http://www.apache.org/licenses/LICENSE-2.0                            *
+///                                                                          *
+/// Unless required by applicable law or agreed to in writing, software      *
+/// distributed under the License is distributed on an "AS IS" BASIS,        *
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. *
+/// See the License for the specific language governing permissions and      *
+/// limitations under the License.                                           *
+///**************************************************************************/
+
+import 'package:optimizely_flutter_sdk/src/data_objects/base_response.dart';
+import 'package:optimizely_flutter_sdk/src/utils/constants.dart';
+
+class FetchQualifiedSegmentsResponse extends BaseResponse {
+  List<String> qualifiedSegments = [];
```

Comment:
We may not need this list. Also ios/android cores won't return qualifedSegments.